cli/cli_app.go

```diff
@@ -217,6 +219,7 @@ GLOBAL OPTIONS:
    terragrunt-source                            Download Terraform configurations from the specified source into a temporary folder, and run Terraform in that temporary folder.
    terragrunt-source-update                     Delete the contents of the temporary folder to clear out any old, cached source code before downloading new source code into it.
    terragrunt-iam-role                          Assume the specified IAM role before executing Terraform. Can also be set via the TERRAGRUNT_IAM_ROLE environment variable.
+   terragrunt-sts-duration                      Session duration for STS session. Can also be set via the TERRAGRUNT_STS_DURATION environment variable.
```

Comment:
NIT: I'm not sure "STS" is going to be as recognizable to many users as "IAM Role." Perhaps `--terragrunt-iam-role-duration` would be a clearer name? This applies to the CLI arg, env var, etc.

Reply:
I agree with the fact that `terragrunt-sts-duration` might be a bit short, however `terragrunt-iam-role-duration` seems "wrong" (i.e it's not really the duration of the IAM role, but of the AssumeRole session)

Do you think `terragrunt-iam-assume-role-duration` / `iam_assume_role_duration` / `TERRAGRUNT_IAM_ASSUME_ROLE_DURATION` might be better?

Reply:
That works for me!